src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/hmi/on_bc_system_capability_updated_notification_from_hmi.cc

```diff
@@ -135,6 +145,48 @@ void OnBCSystemCapabilityUpdatedNotificationFromHMI::Run() {
       }
       break;
     }
+    case mobile_apis::SystemCapabilityType::VIDEO_STREAMING: {
+      if (!system_capability.keyExists(strings::video_streaming_capability)) {
```

Comment:
extract it from switch to function

Reply:
@LuxoftAKutsan Done in bbdf770e46270a5cc6f246d411401e8e0e8bfc4e